migrace/021.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+$this->q("
+
+INSERT INTO r_zidle_soupis (id_zidle, jmeno_zidle, popis_zidle) VALUES
+(20, 'Správce financí GC', 'Právo na přístup do sekce promlčení zůstatků');
```

Comment:
Židle není právo. To je fakt důležité! Zkusil bych se podívat na popisky ostatních židlí a nějak se inspirovat, a kdyby to vůbec nešlo, tak při nejhorším dát prázdný řetězec nebo vykopírovat název židle.

Když to udělám, vychází mi z ostatních židlí třeba "Organizátor, který může nakládat s financemi GC".

Reply:
Vím, že židle není právo. To byl nějaký zkrat.